templates/experiments/components/experiment_actions_column.html

```diff
@@ -9,6 +10,12 @@
      href="{% url type|add:':edit' record.team.slug record.id %}" title="View">
     <i class="fa-solid fa-pencil"></i>
   </a>
+  {% if type == 'chatbots' %}
+    <a class="btn btn-sm"
+       onclick="openCopyDialog({{ record.id }}, '{{ record.team.slug }}')">
```

Comment:
if you give the dialog a consistent ID then you can open it using that ID e.g. `experiment_{{ experiment_id }}_copy_modal.showModal()`